ncm-network/src/main/perl/network.pm

```diff
@@ -265,7 +265,7 @@ sub Configure
     opendir(DIR, $dir_pref);
     # here's the reason why it only verifies eth, bond, bridge, usb and vlan
     # devices. add regexp at will
-    my $dev_regexp='-((eth|seth|em|bond|br|ovirtmgmt|vlan|usb|ib|p\d+p|en(o|(p\d+)?s))\d+|enx[[:xdigit:]]{12})(\.\d+)?';
+    my $dev_regexp='-((eth|seth|em|bond|br|ovirtmgmt|vlan|usb|ib|p\d+p|en(o|(p\d+)?s(?:\d+f)?(?:\d+d)?))\d+|enx[[:xdigit:]]{12})(\.\d+)?';
```

Comment:
The comment above this is now extremely out of date. I am guessing the only correct part left is "add regexp at will". Leaving aside that the regexp is also now at the level of expert, what value is actually adding here, aside from needing to update the code whenever new device types come along? Can we simplify this in anyway?